runtime/cfdumper/main.c

```diff
@@ -516,7 +516,18 @@ static void dumpMethod(J9CfrClassFile* classfile, J9CfrMethod* method)
 	j9tty_printf( PORTLIB, "  Signature: %i -> %s\n", method->descriptorIndex, classfile->constantPool[method->descriptorIndex].bytes);
 	j9tty_printf( PORTLIB, "  Access Flags: 0x%X ( ", method->accessFlags);
 	printModifiers(PORTLIB, method->accessFlags, INCLUDE_INTERNAL_MODIFIERS, MODIFIERSOURCE_METHOD);
-	j9tty_printf( PORTLIB, " )\n");
+	if(classfile->accessFlags & CFR_ACC_VALUE_TYPE) {
+		U_8* nameBytes = classfile->constantPool[method->nameIndex].bytes;
+		if(0 == strcmp((char*)nameBytes, "<vnew>")) {
+			for \(int i = 0; i < classfile->attributesCount; i++) {
+				J9CfrAttribute* attr = classfile->attributes[i];
+				if(attr->tag == CFR_ATTRIBUTE_ImplicitCreation) {
+					j9tty_printf( PORTLIB, " implicit");
+				}
+			}
+		}
+	}
+	j9tty_printf( PORTLIB, ")\n");
```

Comment:
I think this printf can be move to the line after `j9tty_printf( PORTLIB, " implicit");`.

Reply:
OK, I see.